package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "silverstripe-framework",
+  "version": "4.0.0",
+  "description": "The SilverStripe framework",
+  "directories": {
+    "doc": "docs",
+    "test": "tests"
+  },
+  "engines": {
+    "node": "~4.2.0"
```

Comment:
Assuming node follows semver, isn't this too strict? We should require `~4`, so anything lower than 5.x.x.